root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -3435,56 +3832,65 @@ const testData = [
     expected_relationship_type: 'discographyentry',
             expected_clean_url: 'https://ra.co/podcast/491',
        only_valid_entity_types: ['release'],
+        expected_favicon_class: 'residentadvisor-favicon',
   },
   {
                      input_url: 'http://www.ra.co/exchange/552',
              input_entity_type: 'release',
     expected_relationship_type: 'shownotes',
             expected_clean_url: 'https://ra.co/exchange/552',
        only_valid_entity_types: ['release'],
+        expected_favicon_class: 'residentadvisor-favicon',
   },
```

Comment:
I suggest testing the reported URL’s favicon for that bug:

```suggestion
  {
                     input_url: 'http://www.ra.co/exchange/552',
             input_entity_type: 'release',
    expected_relationship_type: 'shownotes',
            expected_clean_url: 'https://ra.co/exchange/552',
       only_valid_entity_types: ['release'],
        expected_favicon_class: 'residentadvisor-favicon',
  },
  {
                     input_url: 'https://www.kenjimiura.com/',
             input_entity_type: 'artist',
    expected_relationship_type: undefined,
            expected_clean_url: 'https://www.kenjimiura.com/',
        expected_favicon_class: 'no-favicon',
  },
```

Reply:
Sounds sensible, will add